tests/schemas/invalid/test_invalid_calculated_summary_answer_ranges.json

```diff
@@ -0,0 +1,171 @@
+{
+  "mime_type": "application/json/ons/eq",
+  "language": "en",
+  "schema_version": "0.0.1",
+  "data_version": "0.0.3",
+  "survey_id": "0",
+  "title": "A test schema to demo Calculated Summary",
+  "theme": "default",
+  "description": "A schema to showcase Calculated Summary page and usage in value source.",
```

Comment:
Worth updating to explain why this schema is invalid

Reply:
Updated description 👍